batch2/batch/worker.py

```diff
@@ -95,7 +95,8 @@ def __init__(self, spec, pod, log_directory):
             'Cmd': self.spec['command'],
             'Image': self.spec['image'],
             'HostConfig': {'CpuPeriod': 100000,
-                           'CpuQuota': self.cores_mcpu * 100}
+                           'CpuQuota': self.cores_mcpu * 100,
+                           'Binds': None}
```

Comment:
What's this do?

Reply:
I thought maybe extra volumes were being created if the default value was `[]` rather than `None`. I couldn't replicate the behavior locally, so I'm deleting it.